app/controllers/response_sets_controller.rb

```diff
@@ -46,6 +47,22 @@ def create
     end
   end
 
+  def transfer_table
+    @response_set.coded = @parent_set.coded
+    @response_set.oid = @parent_set.oid
+    @response_set.description = @parent_set.description
+    @response_set.name = @parent_set.name
+    @parent_set.responses.each do |r|
```

Comment:
I'm not certain yet if we want to copy all of the responses over.  It may be better to leave them where they are and simply reference them through the parent but it's something that we need to think about.